data/pokedex.ts

```diff
@@ -12110,6 +12110,8 @@ export const BattlePokedex: {[speciesid: string]: SpeciesData} = {
 		weightkg: 215,
 		color: "Blue",
 		eggGroups: ["Undiscovered"],
+		cosmeticFormes: ["Xerneas-Neutral"],
```

Comment:
Xerneas-Neutral is not a cosmetic forme, because PS defines cosmetic formes as having no legality differences, while Xerneas-Neutral is illegal in battles.

The easiest solution would be just to have the forme named in `formeOrder` but not otherwise mentioned, like this:

```suggestion
```

But a more thorough solution might be to make it an illegal forme in some way, possibly marking it as `Unobtainable`?

Reply:
While we're at it, we probably want to set `baseForme: "Active",`

Reply:
I think the latter option is preferable, the test I wrote (which can be modified of course, but I feel its fairly useful for testing the constraints and avoid typos) expects that the forme is formeOrder is a real species